### DISCUSSION TITLE
Supabase example

### DISCUSSION THREAD

**Discussion by nsantos16** [blitz-js/blitz#2681] [Ideas]
Hi everybody 👋

I'm wondering if you think that is useful to add an example of [Supabase](https://supabase.io/) to the example folder as well.

This was an [issue](https://github.com/supabase/supabase/issues/151) on Supabase Repo and I want to double check with you if you think it is useful, and if that was the case I offer to carry out this example.

Waiting for your comments 😄

**Comment by flybayer** [comment]
Hey sure, that'd be great!

**Comment by manucoffin** [comment]
That would be very useful! Especially for the auth part, because it's quite complicated to combine supabase with the blitzjs auth management

**Comment by phillipphoenix** [comment]
Any news on this?

**Reply by ilangleben19** [reply]
Bump
